src/frontend/apps/standalone_site/.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  parser: '@typescript-eslint/parser', // Specifies the ESLint parser
+  extends: [
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended', // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'plugin:prettier/recommended',
+  ],
+  parserOptions: {
+    ecmaVersion: 2018, // Allows for the parsing of modern ECMAScript features
+    sourceType: 'module', // Allows for the use of imports
+    ecmaFeatures: {
+      jsx: true,
+    },
+    tsconfigRootDir: __dirname,
+    project: ['./tsconfig.json'],
+  },
+  rules: {
+    '@typescript-eslint/explicit-module-boundary-types': 0,
+    '@typescript-eslint/no-floating-promises': 0,
+    '@typescript-eslint/no-unused-vars': ['error', { varsIgnorePattern: '^_' }],
+    '@typescript-eslint/explicit-function-return-type': 0,
+    '@typescript-eslint/no-non-null-assertion': 'error',
+    'array-callback-return': 'error',
+    'block-scoped-var': 'error',
+    curly: 2,
+    'default-case': 'error',
+    eqeqeq: 2,
```

Comment:
Can you apply the change made in this commit to this eslint config ? https://github.com/openfun/marsha/commit/40bfc69ff1827a866a238b5b1b7cb29be818ce0a